extensions/ql-vscode/src/databases.ts

```diff
@@ -470,6 +476,15 @@ export class DatabaseItemImpl implements DatabaseItem {
     return uri.scheme === zipArchiveScheme &&
       decodeSourceArchiveUri(uri).sourceArchiveZipPath === this.sourceArchive.fsPath;
   }
+
+  /**
+   * Whether the database may be affected by test execution under the given path. It returns true
+   * if the database is backed by a testproj directory at or underneath the given path.
+   */
```

Comment:
It's not useful to duplicate the comment here. Mostly, I'm concerned about one of these copies going out of date.